### ISSUE TITLE
React children of SbEditable that are short-circuit rendered throw an Error

### ISSUE THREAD

**Issue by cole-kerr-ef** [storyblok/storyblok-react#3]
**Source component:**
![image](https://user-images.githubusercontent.com/51664758/59206677-727ff880-8b73-11e9-9ad4-eee449852c29.png)

**As used in SbEditable:**
![image](https://user-images.githubusercontent.com/51664758/59205940-e6210600-8b71-11e9-9566-416994ddfb84.png)

When props.iconImgSrc is not passed, I want my component to be able to short-circuit render as false. 
I cannot do this as long as it is a direct child of SbEditable, because SbEditable does not check to see if the result from _ReactDOM.findDomNode(this)_ is an object, and attempts to access method property .setAttribute in line 18 of the source code. 

I made a bugfix branch adding a simple 
`if (!(el instanceof Object)) { return }`, but am unable to push up to a branch in order to create a PR (I'm a bit knew to open-source contribution, sorry!)

**Comment by onefriendaday** [comment]
So you would like to render nothing if iconImgSrc is not passed? Couldn't you just check for the iconImgSrc before rendering SbEditable?

Another option would be to add a dom element as wrapper.

~~~jsx
<SbEditable content={props}>
<span>
  <ContentLabelIcon />
</span>
</SbEditable>
~~~

I also created a PR for the check of the type. Does this work for you?:
https://github.com/storyblok/storyblok-react/pull/4/files

**Comment by cole-kerr-ef** [comment]
@onefriendaday The PR solution works great, it's basically what I was going for. Thanks a ton!

**Comment by onefriendaday** [comment]
Published in version 0.0.9
